### PR TITLE
Drop PPC builds

### LINKED ISSUE #187: Disable aarch & ppc builds until new build system arrives
Currently, the builds on drone and aarch time out consistently, and cross-compiling on azure times out for aarch (see #178) and fails for PPC with #186, resp. stuff like scipy/scipy#14560.

It's currently very painful to maintain (3h+ waiting times for each run, several restarts necessary), and considering the download numbers (see #171), I don't think it's worth to carry this "baggage" along while we try to solve other things (like #160, #177).

I'd suggest to disable aarch & ppc builds at least until the new build system comes along (see #164; scheduled for scipy 1.8.0 in ~December).

CC @conda-forge/scipy @isuruf

PS. If necessary, we could still do one-off builds for eventual releases in the 1.7.x branch; my proposal is just to not keep it running for every PR.

### PR DESCRIPTION
**Pull request by h-vetinari**
As discussed in #187, the aarch/ppc builds were causing lots of trouble (not least that PPC has been silently failing the test suite for a while, see #186), take forever to build, and have an absurd ratio of maintenance-effort vs. usage numbers (see #171).

We disable them for the time being, in the hope that meson builds (#164) will have less issues. As mentioned in #187, we can create a `v1.7` branch if we need to come back to these builds for the 1.7 series.

CC @conda-forge/core @rgommers for visibility.

Closes #187

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -66,11 +66,8 @@ test:
     # show SIMD features (some failures occur depending on presence/absence of e.g. AVX512)
     # change to np.show_config() for v>=1.22.0, cf. numpy/numpy#19130
     - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
-    # Run only fast tests for key modules on drone as it times out; otherwise set label='full' (note default: label='fast')
-    {% set label = "'fast'" if aarch64 else "'full'" %}
-    {% set tests = "['scipy.linalg', 'scipy.fft']" if aarch64 else "None" %}
     # for signature, see https://github.com/scipy/scipy/blob/v1.7.0/scipy/_lib/_testutils.py#L27
-    - python -c "import scipy; scipy.test(verbose=2, label={{ label }}, tests={{ tests }}, extra_argv=['--durations=50'])"
+    - python -c "import scipy; scipy.test(verbose=2, label='full', extra_argv=['--durations=50'])"
```
[the remark this answers is not in the document]
Comment (h-vetinari) [reply recipe/meta.yaml:70]: It's fine for everything but aarch...

**Comment by h-vetinari** [comment]
> I'm a little surprised about the low number for aarch64

Do you mean the download numbers? I took your 👍 from #187 as a sign of support for this, but (aside from the PPC builds not running the test suite; which I had managed to forget myself...), it's not _that_ urgent to act ATM.

**Comment by h-vetinari** [comment]
@rgommers, aside from mamba-builds not being allowed yet, #192 is still hitting the timeout on aarch, and travis is just failing at the most basic task (downloading sources from github, see https://github.com/conda-forge/conda-forge.github.io/issues/1521).

I think a reasonable approach would be to keep aarch running on azure (mamba or not), but PPC has got to go under the current conditions (especially the state of travis + #186).

**Comment by jaimergp** [comment]
Check their Gitter. I think it's the same issue: https://gitter.im/QuantStack/Lobby?at=61692853ee6c260cf71ce724

**Comment by h-vetinari** [comment]
Obsoleted (hopefully) by #193

**Pull request closed without merging**